starky/src/compressor12/plonk_setup.rs

```diff
@@ -708,7 +710,7 @@ pub fn plonk_setup_compressor(
                 w * ks[j - 1],
             );
         }
-        w = w * (crate::constant::MG.0[plonk_setup_info.n_bits].to_be());
+        w = w * (F3G::from(crate::constant::MG.0[plonk_setup_info.n_bits]).to_be());
```

Comment:
`to_be` is actually to return MG.0, here we do not have to convert it to F3G and then extract it again.